Makefile

```diff
@@ -24,6 +24,14 @@ export TF_IN_AUTOMATION="true"
 export TF_VAR_MATURITY=${MATURITY}
 export TF_VAR_DEPLOY_NAME=${DEPLOY_NAME}
 PYTHON_VER ?= python3
+
+
+REPO_DIR := /CIRRUS-DAAC
+CIRRUS_DAAC_BRANCH := $(shell git -C $(REPO_DIR) rev-parse --abbrev-ref HEAD)
+CIRRUS_CORE_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+export CIRRUS_CORE_VERSION=${CIRRUS_CORE_BRANCH}
+export CIRRUS_DAAC_VERSION=${CIRRUS_DAAC_BRANCH}
```

Comment:
I am seeing some interesting behavior pulling the version from the branch. I may need to go a different route.